www-apps/kibana-bin/kibana-bin-6.2.4.ebuild

```diff
@@ -30,20 +30,20 @@ src_prepare() {
 	default
 
 	# remove bundled nodejs
-	rm -rv node || die
+	rm -r node || die
+
+	# remove empty unused directory
+	rm -r data || die
 }
 
 src_install() {
-	appdirs="/var/log/${MY_PN} /var/lib/${MY_PN} /var/cache/${MY_PN}"
-	keepdir $appdirs
-	fowners kibana: $appdirs
-	fperms 0750 $appdirs
-
-	rm -rv data || die
+	diropts -m 0750 -o ${MY_PN} -g ${MY_PN}
```

Comment:
Maybe move this to the end of the function, so you wouldn't have to clean `diropts`.

Reply:
Thanks :)